src/pages/workspaces/workspace/Data.js

```diff
@@ -557,21 +557,29 @@ const WorkspaceData = _.flow(
                 ])],
                 () => div({ role: 'list', style: { fontSize: 14, lineHeight: '1.5' } }, [
                   _.map(([tableName, { count }]) => {
+                    const canCompute = !!(workspace?.canCompute)
                     return h(DataTypeButton, {
-                      buttonStyle: { borderBottom: 0, height: 40 },
+                      buttonStyle: canCompute ? { borderBottom: 0, height: 40 } : { borderBottom: 0, height: 40, color: colors.dark(0.25) },
```

Comment:
Option for a bit more concision:
```suggestion
                      buttonStyle: { borderBottom: 0, height: 40, ...(canCompute ? {} : {color: colors.dark(0.25)}) },
```